documents/entity/service.go

```diff
@@ -265,22 +262,30 @@ func (s service) GetVersion(ctx context.Context, documentID []byte, version []by
 	return s.get(ctx, documentID, version)
 }
 
-func (s service) GetEntityByRelationship(ctx context.Context, entityIdentifier, relationshipIdentifier []byte) (documents.Model, error) {
+func (s service) GetEntityByRelationship(ctx context.Context, relationshipIdentifier []byte) (documents.Model, error) {
+	model, err := s.erService.GetCurrentVersion(ctx, relationshipIdentifier)
+	if err != nil {
+		return nil, entityrelationship.ErrERNotFound
+	}
+
+	relationship := model.(*entityrelationship.EntityRelationship)
```

Comment:
maybe put a relationship, ok check here in case casting the model to EntityRelationship fails (if an entityID is used instead of a relationshipID, for example)